app/src/main/java/org/mifos/mobile/ui/beneficiary/presentation/icons.kt

```diff
@@ -0,0 +1,101 @@
+package org.mifos.mobile.ui.beneficiary.presentation
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mifos.mobile.R
+import org.mifos.mobile.core.ui.theme.MifosMobileTheme
+
+
+/***
+ *this is a composable that is made up of multiple icons from the res folder,
+ * the composable takes in parameters such as
+ * @param[addIconclicked] for taking actions when the add icon is clicked
+ * @param[scanIconClicked]  for taking icons when the scan icon is clicked
+ * @param[uploadIconClicked] for taking actions when the upload icon is clicked
+ * @param[modifier] for modifying the composable.
+ * Reference the preview function to see a preview of the screen
+ *
+ * */
```

Comment:
also there is no need to write a java doc when the name of the parameters are self explanatory

Reply:
I have made this corrections